schema/me/consignments/add_asset_to_submission_mutation.js

```diff
@@ -0,0 +1,32 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { AssetType } from "./asset"
+
+export default mutationWithClientMutationId({
+  name: "AddAssetToConsignmentSubmission",
+  description: "Attach an impulse asset to a consignment submission",
+  inputFields: {
+    asset_type: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The type of the asset, usually 'image'",
+    },
+    gemini_token: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The id of the artist",
```

Comment:
This field and the below one seem to have inaccurate descriptions.

Reply:
Thanks!